scenes/quests/lore_quests/quest_002/2_grappling_hook/intro_start.dialogue

```diff
@@ -0,0 +1,9 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+~ start
+Matilda: Go take the thread, it will teleport you to the town.
+do animation_player.play(&"eat_floor")
+do animation_player.animation_finished
+Matilda: Hmmm.. the Void is blooming fast here.
+Matilda: You’d only be able to reach the town using a [b]grappling hook[/b]. That's how I managed to survive. Take mine.
```

Comment:
Nice idea to embolden the text! However it doesn't work on the web platform:

<img width="819" height="672" alt="Image" src="https://github.com/user-attachments/assets/491b1901-04f0-48fb-88ab-adcff02314c8" />

presumably because we don't have a bold-weight font.

Maybe we can (later) change this to use colour instead?